core/src/main/java/org/infinispan/globalstate/ScopedPersistentState.java

```diff
@@ -50,4 +50,9 @@
     * Performs the specified action on every entry of the state
     */
    void forEach(BiConsumer<String, String> action);
+
+   /**
+    * Returns the checksum of the common properties (i.e. those not prefixed with @)
```

Comment:
Sorry, I need to reword that: the @ props are those that need to be excluded from the checksum computation (e.g. timestamp, version)